src/menu.cc

```diff
@@ -112,6 +112,16 @@ Menu::Menu() {
         </item>
       </section>
       <section>
+      <item>
+        <attribute name='label' translatable='yes'>_Toggle _Line _Comment</attribute>
+        <attribute name='action'>app.source_comments_toggle</attribute>
+      </item>
+      <item>
+        <attribute name='label' translatable='yes'>_Toggle _Block _Comment</attribute>
```

Comment:
Also not sure if one should move this menu item from the Source menu, as Add Documentation is related (doxygen comments) to Toggle Comments.